packages/jaeger-ui/src/components/App/index.jsx

```diff
@@ -92,6 +93,9 @@ export default class JaegerUIApp extends Component {
             <Router history={history}>
               <Page>
                 <Switch>
+                  <Route path="/demo">
+                    <Demo />
+                  </Route>
```

Comment:
make sure to clear up this before final PR

Reply:
Yeah, this PR is too cluttered and only made for you and @yashrsharma44 to view progress, I'll make another PR with only required changes